### DISCUSSION TITLE
Detecting motion at night from car headlights

### DISCUSSION THREAD

**Discussion by noefingway** [blakeblackshear/frigate#14051] [Ask A Question]
I've been using Frigate for several years with multiple cameras on my farm. Works great during daylight hours but it won't pickup car headlights. My driveway is 900' long, I have a camera pointed at a length of the driveway, but vehicles with headlights on are not detected. This is in the "it's annoying" category more than anything else since there is rarely anyone coming down the driveway at night and my Unifi camera isn't good enough to pick up a tag. So any thoughts?

BTW - the infrared detection works great on insects flying in front of the camera but can't detect any of the skunks, foxes or coyotes wandering up and down the lane at night.

**Comment by NickM-27** [comment]
it's difficult because two bright lights isn't really "a car" like an actual car looks like. It's possible Frigate+ models would be better and you could train your model to detect the headlights as a car.

**Reply by Davy-H** [reply]
What I've ended up doing is using the audio detection to get vehicles at night. And you need to include all sorts of weird sounds

        - tire_squeal
        - skidding
        - toot
        - honk
        - car
        - motor_vehicle
        - truck
        - yell
        - scream
        - scratching
        - frog
        - speech
        - vehicle
        - stomach_rumble
        - whoosh
        - explosion
        - roaring_cats

**Comment by noefingway** [comment]
Thanks for the tip on audio - I don't regularly keep up with new developments, so I wasn't aware that this feature existed.
